examples/videocomposer/README.md

```diff
@@ -2,6 +2,138 @@
 
 MindSpore implementation & optimization of [VideoComposer: Compositional Video Synthesis with Motion Controllability](https://arxiv.org/pdf/2306.02018.pdf).
 
+## Gallery
+
+### Inference different conditions from a video
+<p align="center">
+<img src=assets/y-depth_vid0003.gif width="720" />
```

Comment:
why not upload as other gifs using git content url

Reply:
It's a draft. Still trying different ways to make the gif loop infinitely. Will use the git content url.

Reply:
There's no issue with playing a GIF indefinitely on GitHub:
https://github.com/mindspore-lab/mindone/blob/master/examples/svd/README.md
When you save the GIF, you need to specify the `loop` parameter for it to play indefinitely.